### PR TITLE
USB: wpanusb: Correct configurations and add sanitycheck config

### PR DESCRIPTION
**Pull request by finikorg**

### CODE REVIEW

File: samples/net/wpanusb/overlay-mcr20a.conf [hunk 1]
```diff
@@ -1,2 +1,4 @@
+CONFIG_NET_L2_ETHERNET=n
+CONFIG_SPI=y
```
Comment (tbursztyka) [inline samples/net/wpanusb/overlay-mcr20a.conf]: I don't think you need this one, it's already selected by mcr20a

Comment (finikorg) [reply samples/net/wpanusb/overlay-mcr20a.conf]: That is a problem, it is not selected. If you apply first sanitycheck patch you get error with "sanitycheck -t usb"

File: samples/net/wpanusb/sample.yaml [hunk 2]
```diff
@@ -6,3 +6,13 @@ tests:
     harness: net
     depends_on: ieee802154 usb_device
     tags: net ieee802154 usb
+  test_wpanusb_frdm_cr20a:
+    harness: net
```
Comment (jukkar) [inline samples/net/wpanusb/sample.yaml]: Could you add the "harness:" and "tags:" lines under "common:" block, these two tags are now unnecessarily triplicated in this file.

**Review by jukkar — CHANGES_REQUESTED** [review]

File: samples/net/wpanusb/sample.yaml (hunk 2 above)
Comment (finikorg) [reply samples/net/wpanusb/sample.yaml]: This is done now.

**Review by masz-nordic — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Pull request merged**